hotham/src/components/stage.rs

```diff
@@ -1,39 +1,32 @@
-/// Marker component for the player's frame of reference
+/// A component to assist with artificial locomotion.
+///
+/// On startup, [`crate::Engine`] will create an entity with the [`Stage`] component, and an entity with the
+/// [`super::HMD`] component (representing the user's headset), parented to the stage entity.
+///
+/// This makes moving the player around in the virtual world *relatively* straightforward, as all is required is
+/// to move the [`Stage`]. The engine will ensure that the player's position in the game simulation AND the
+/// virtual cameras are correctly updated to account for this.
+///
+/// In short, the final position of the player in the game simulation (ie. global space) is:
+///
+/// `stage.position * hmd.position`
```

Comment:
I am a bit uncertain about what "position" should be replaced with here. "pose" or "transform"?